surfer/utils.py

```diff
@@ -32,7 +32,7 @@ class _MayaviThresholdPatch(object):
     In Mayavi 4.5, filters seem to be missing a .point_data attribute that
     Threshold accesses on initialization.
     """
-    need_patch = LooseVersion(mayavi.__version__) <= LooseVersion('4.5.0')
+    need_patch = LooseVersion(mayavi.__version__) == LooseVersion('4.5.0')
```

Comment:
It seemed to be fixed on master, so I'm assuming it will be fixed > 4.5. I don't know about mayavi 4.4, I tried reinstalling 4.4 with conda to try it out but hit upon error after error suggesting backend compatibility issues and gave up... :( does anybody still have a working 4.4 env?